### PR TITLE
fix: 94 - [tour] restored white background color

### LINKED ISSUE #94: Tour - Columns section in the home page should have a 100% width and white background
The section itself should occupy the whole viewport, otherwise the background image of the hero is displayed as background

Screenshot 2023-05-24 at 16 30 23

### PR DESCRIPTION
**Pull request by etomaello**
This PR will fix the issue that shows the hero background image between sections in the tour homepage.

Fix #94

Test URLs:
- Before: https://main--realmadrid--hlxsites.hlx.page/tour-bernabeu
- After: https://94-full-columns-background-fix--realmadrid--hlxsites.hlx.page/tour-bernabeu

### CODE REVIEW

File: blocks/badge/badge.js [hunk 1]
```diff
@@ -22,4 +35,9 @@ export default function decorate(block) {
     block.prepend(anchorWrapper);
   }
   styleBlock(block);
+  // extract merge-blocks-desktop elements in a dedicated div
```
Comment (chicharr) [inline blocks/badge/badge.js:38]: Adding this logic in the `badge` block, does it mean that the `merge-blocks` feature would only work when there is a badge?

File: blocks/badge/badge.js [hunk 2]
```diff
@@ -22,4 +35,9 @@ export default function decorate(block) {
...
+  const mergeBlocks = getParents(block, '.merge-blocks-desktop');
```
Comment (chicharr) [inline blocks/badge/badge.js:39]: isn't this the same as: `block.closest('.merge-blocks-desktop')`

File: blocks/badge/badge.js (hunk 1 above)
Comment (etomaello) [reply blocks/badge/badge.js:38]: @chicharr that's true, where can I move this function to be available to the other templates?

File: blocks/badge/badge.js (hunk 2 above)
Comment (etomaello) [reply blocks/badge/badge.js:39]: @chicharr my bad, I will change that. thanks for reporting this

File: blocks/badge/badge.js (hunk 1 above)
Comment (chicharr) [reply blocks/badge/badge.js:38]: If we want to make it a global feature it should be in the scripts.js

File: blocks/badge/badge.js (hunk 2 above)
Comment (etomaello) [reply blocks/badge/badge.js]: @chicharr done, I lef the getParents function in the scirpts.js for future use cases. Let me know if I must remove it. thanks

File: blocks/badge/badge.js (hunk 1 above)
Comment (etomaello) [reply blocks/badge/badge.js:27]: done

**Pull request merged**